src/Components/index.ts

```diff
@@ -0,0 +1,9 @@
+export { Home } from "./Home";
```

Comment:
Alternative, you can write 

export * from "./Home". This is helpful when you have multiple functions in a file. (not for this case, just mentioning)

Reply:
Oh, I wasn't aware of this. Thanks, it would be useful in another project of mine 🙂